### ISSUE TITLE
In dark theme the "Add" button in keyboard shortcuts UI is not readable

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#16801]
## Description

In dark theme the "Add" button in keyboard shortcuts UI is not readable

## Reproduce

1. Change theme to "JupyterLab Dark"
2. Open Settings
3. Open "Keyboard Shortcuts"
4. Hover over a row
5. See

![image](https://github.com/user-attachments/assets/4449a329-c853-4f93-8b97-a0d22aac79c1)

## Context

- Browser and version: Chrome
- JupyterLab version: 4.3.0b1

**Comment by krassowski** [comment]
The existing shortcut itself when edited has poor contrast:

![Uploading image.png…]()

**Comment by Gannu456** [comment]
Can i work upon this, if it is still open?

**Comment by krassowski** [comment]
Yes, it is still open, please do open a PR :)

**Comment by omeizan** [comment]
Hi, I'm not sure if this has been worked on yet, can I work on this?

**Comment by krassowski** [comment]
Yes, you can. No one opened a PR so far.

**Comment by Madhu3456** [comment]
Hi, after trying some permutations of colors I came up with this, can someone verify?

![Screenshot from 2024-12-10 21-28-46](https://github.com/user-attachments/assets/0f12ac9c-f9be-4d50-83fa-1cd7b4a41106)


---On hover
![Screenshot from 2024-12-10 21-28-49](https://github.com/user-attachments/assets/9b0465bf-6e79-4ec9-9d89-12d0a67646aa)

**Comment by krassowski** [comment]
Is the top one the normal state? If so the contrast looks too weak to me. We aim for WCAG AA contrast you can calculate it https://webaim.org/resources/contrastchecker/

**Comment by Madhu3456** [comment]
Yes, the top one is the normal state and I feel it's weak as well. I will use the resource.

**Comment by Madhu3456** [comment]
To add a color(--md-blue-700) that is brighter than "--jp-brand-color0"(--md-blue-900) should I change the exiting variable names from 0 to 1, 1 to 2 and so on and mark the new color as --jp-brand-color0 ? If so I'll also have to make changes in all the places these are used?

https://github.com/jupyterlab/jupyterlab/blob/2607c1093539850b5b618352dc94893a8b662e48/packages/theme-light-extension/style/variables.css#L219

**Comment by krassowski** [comment]
> To add a color(--md-blue-700) that is brighter than "--jp-brand-color0"(--md-blue-900) should I change the exiting variable names from 0 to 1, 1 to 2 and so on and mark the new color as --jp-brand-

No, this would have unintended consequences as all the colours would shift and extensions/themes would need to change as well - I think not a good idea. Changing something which only affects this single button would be best

**Comment by Gannu456** [comment]
Is this issue still open..can you assign it to me?
